BrandonBeltran/hackerrank/03/JavaSort.java

```diff
@@ -0,0 +1,69 @@
+package bootcamp;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Scanner;
+
+class Student implements Comparable<Student>{
```

Comment:
Comparable!! [clapping]
But implements compareTo just for one double field ?? Just let's keep that in mind for future coding